src/sdc11073/pysoap/soapclient.py

```diff
@@ -183,6 +182,7 @@ def _mk_http_connection(self) -> [HTTPSConnectionNoDelay, HTTPConnectionNoDelay]
 
     def connect(self):
         """Connect to netloc."""
+        self._has_connection_error = False
```

Comment:
does it make sense to use `threading.Event` instead of a `bool`?

Reply:
no, I would use threading.Event for thread sync stuff, but this is only a internal variable. Keep it as simple as possible.